app/services/foreman/access_permissions.rb

```diff
@@ -317,7 +317,7 @@
                                       :externalNodes, :pxe_config, :storeconfig_klasses, :auto_complete_search, :bmc,
                                       :runtime, :resources, :templates, :overview, :nics],
                                     :dashboard => [:OutOfSync, :errors, :active],
-                                    :unattended => [:template, :provision],
+                                    :unattended => [:template, :provision] + TemplateKind.all.map(&:name).map(&:to_sym),
```

Comment:
also I'd prefer using some hard list (maybe just put this into constant https://github.com/theforeman/foreman/blob/develop/db/seeds.d/07-config_templates.rb#L9), 

* one reason is that if plugin introduces new kind it should also map it's own permission, this would allow to review any custom template kind with view_host permission
* second is that this is required in initializer https://github.com/theforeman/foreman/blob/develop/config/initializers/foreman.rb#L1 and it's good to avoid using AR there